test/src/us/kbase/kbasefeaturevalues/test/KBaseFeatureValuesServerTest.java

```diff
@@ -258,11 +268,26 @@ public void testMainPipeline() throws Exception {
         Assert.assertEquals(2, clSet3.getFeatureClusters().get(1).getIdToPos().size());
         Assert.assertTrue(clSet3.getFeatureDendrogram().startsWith("("));
         Assert.assertTrue(clSet3.getFeatureDendrogram().endsWith(");"));
+
+
+	List<Tuple11<Long,String,String,String,Long,String,Long,String,String,Long,Map<String,String>>> info = getWsClient().saveObjects(new SaveObjectsParams().withWorkspace(testWsName)
```

Comment:
This is another case of saving an object to the ws, getting it, and then comparing what you saved to what you got. It doesn't seem to test feature values code at all.

There's more places like this in the code, but I'm going to stop at 2 comments.